packages/actionmenu/src/react/item.tsx

```diff
@@ -43,99 +47,107 @@ const styles = {
   textOnly: () => css(stylesheet['.psds-actionmenu__text-only'])
 }
 
-interface Props extends Omit<HTMLAttributes<HTMLLIElement>, 'onClick'> {
-  disabled?: boolean
-  tagName?: string
+interface ItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'onClick'> {
   className?: string
-  origin?: string
-  onClick?: (event: React.MouseEvent, value: string | number) => void
+  disabled?: boolean
+  href?: string
   nested?: React.ReactNode
+  onClick?: (event: React.MouseEvent, value: string | number) => void
+  origin?: string
+  tagName?: string
   value?: string | number
```

Comment:
sorry i can't help but alphabetize out of habit.

i did add `href` - it's something we missed the first time around but the stories helped find it